docs_user/main.adoc

```diff
@@ -11,6 +11,10 @@ include::adoption-attributes.adoc[]
 
 include::assemblies/assembly_rhoso-180-adoption-overview.adoc[leveloffset=+1]
 
+ifeval::["{OpenStackPreviousInstaller}" == "director_operator"]
+include::assemblies/assembly_prepare_director_operator_for_adoption_process.adoc[leveloffset=+1]
```

Comment:
I think we're putting all OSPdO content in its own chapter at the end of the adoption guide. I can take care of the reorganization later.

Reply:
ok